homeassistant/components/nordpool/config_flow.py

```diff
@@ -53,17 +59,18 @@ async def test_api(hass: HomeAssistant, user_input: dict[str, Any]) -> dict[str,
     """Test fetch data from Nord Pool."""
     client = NordPoolClient(async_get_clientsession(hass))
     try:
-        data = await client.async_get_delivery_period(
+        await client.async_get_delivery_period(
             dt_util.now(),
             Currency(user_input[CONF_CURRENCY]),
             user_input[CONF_AREAS],
         )
+    except NordPoolAuthenticationError:
+        return {"base": "invalid_auth"}
```

Comment:
Auth?

Reply:
I put it here in case they will change something so that it then pops up correctly.
There is only a certain amount of historical data you can see without logging in.
I update the string so I hope then it makes more sense (don't expect it but you never know if they will change something)